src/global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why is this? Nemělo by to být v .gitignore?

Reply:
Myslím že ne? Je to nastavení projektu, který .NET SDK použít

Reply:
To by ale měl brát z .csproj souboru ne?

Reply:
Idk, můj dotnet cli se furt snažil načíst verzi 6 (co mám nainstalovanou)